### ISSUE TITLE
[MU4 Issue]  Installing Musescore 4 crushed Musescore 3

### ISSUE THREAD

**Issue by RFycard** [musescore/MuseScore#14024]
**Describe the bug**
Big trouble, installing Musescore 4 crushed Musescore 3 for me!!!
And yet I was careful to close the 3 before opening the 4 (I should not have, apparently!!!)
Nevertheless, I was able to reinstall Musescore 3.6.2
The 2 work in parallel, but you have to assign the opening of .mscz and .mscx to MS 3 while 4 is only a beta!!!

 - OS: macOS Monterey 12.6 MacBook Pro M1 Pro 16"

**Comment by cbjeukendrup** [comment]
Do you mean that installing MuseScore 4 _via the Muse Hub_ uninstalls MuseScore 3? That should not happen indeed.

> you have to assign the opening of .mscz and .mscx to MS 3 while 4 is only a beta!!!

On macOS, we as developers don't have any control over this. macOS itself decides which app it uses to open a file; you can customise this in the "Info" dialog for the file in Finder.

**Comment by RFycard** [comment]
> Do you mean that installing MuseScore 4 _via the Muse Hub_ uninstalls MuseScore 3? That should not happen indeed.
> 
Yes it wiped me out Musescore 3, in 30 years on Mac I had never seen anything like it!!!

**Comment by shadowphiar** [comment]
Similar experience on Mac OS 12.6.

I actually still had Musescore 3 open when I installed Muse Hub and, through it, MuseScore 4. MuseScore 3 was removed from the applications folder, and the running icon in the dock was replaced with a question mark! 

Reinstalling 3.6.2 has been fine, but really, when the changes between major versions are so fundamental it needs to remain possible to run both in parallel for a while (especially while 4 is still in beta!)
